### PR TITLE
Makes NULL crates cost 35,000 and contains a uplink implant, uplink implant now contains 14 TC

### PR DESCRIPTION
**Pull request by ma44**
no lynch pls 

Also you coders can go leave this open if you think the infinite money exports isn't fixed, merge some other PRs like the ATM and see how it goes.

### CODE REVIEW

**Comment by optimumtact** [comment]
I am absolutely 100 percent against introducing ways to access more traitor points outside of the roundstart balance given to traitors as it causes too many balance and salt memes.

Especially in a system that has it's balance so broken (cargo and cargo exports)

it's a :-1: from me.

**Comment by ma44** [comment]
@optimumtact We can wait for the cargo exports to be unbroken

**Comment by ShadowDeath6** [comment]
can you still buy uplink implants?
if so then just
>buy uplink implant
gain 4 tc

**Comment by ma44** [comment]
Uplink implants cost 14 TC, this PR makes it so you can spend 14 TC instead of 10TC

**Comment by PKPenguin321** [comment]
super ultra mega :-1: 
half the fun of null crates is not knowing what you're going to get
nerfs uplink implants for a shitty reason
lets cargo get infinite traitor items they can pull out of their asses at pretty much any time

really awful

**Comment by ShadowDeath6** [comment]
>uplink implant not giving equal tc worth spent on it 

why does it exist again? storage implant already fills its niche for less considering you can shove a box full of tools plus your PDA with more TC than the uplink implant's worth

**Comment by ma44** [comment]
@PKPenguin321 This isn't a nerf to uplink implants, you can still buy it for 14 TC
Also some kits in NULL crates give a lot more than 14 TC and sometimes gives special stuff like tele powers from the dark lord kit

**Comment by ShadowDeath6** [comment]
the kits are intended to be worth 20 TC

if you wanna make it op go all in and make it have the syndie surplus crate or some shit, obviously adjust price to fit the cost of it

**Comment by optimumtact** [comment]
the uplink implant is stealthy, that's it's benefit

**Comment by Firecage** [comment]
Bad ideal. Even with the point cost increase, this would be a big buff to Null Crates, especially considering how broken Goofcargo is, this increase to the cost doesnt matter much.

It should still be a random bundle, since, You can either get an absolute shit-tier bundle, or you can get a good bundle like the Sith lord one. With a Uplink implant, you will be able to pick and choose what to buy. 

So basically, point increase which doesnt matter with broken cargo, removes randomness from the Null Crate and making it so people can instead choose what to buy.

**Comment by kevinz000** [comment]
lynch. no. why ruin cargo man.

**Comment by Dorsisdwarf** [comment]
@ShadowDeath6 Uplink implants used to be 20TC before I got to them in my Great Price Un-Fuckening

**Comment by KorPhaeron** [comment]
unpopular
